imo2001_q6.lean

```diff
@@ -0,0 +1,125 @@
+/-
```

Comment:
You should delete this file, since you have a copy in the `archive/imo` folder already.